test/base/WithInjector.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package base
+
+import com.codahale.metrics.SharedMetricRegistries
+import play.api.inject.guice.GuiceApplicationBuilder
+
+trait WithInjector {
+
+  SharedMetricRegistries.clear()
+
+  val injector = GuiceApplicationBuilder().injector()
```

Comment:
I think some helper methods could be useful here,

with `MockitoSugar` we get `mock[T]`, maybe a method that follows the same pattern e.g. `real[T]` or just `instanceOf[T]` so we can declutter the tests a little? What do you think?

Reply:
Done, changes in separate commit :)